src/Calculator/Views/Calculator.xaml.h

```diff
@@ -100,6 +103,18 @@ public
         bool m_IsLastFlyoutMemory = false;
         bool m_IsLastFlyoutHistory = false;
 
+        // Used for disabling/enabling behavior involving AlwaysOnTop mode
+        bool ClearMemoryButtonErrorPrevState;
+        bool MemRecallErrorPrevState;
+        bool MemPlusErrorPrevState;
+        bool MemMinusErrorPrevState;
+        bool MemButtonErrorPrevState; 
+        bool ClearMemoryButtonEnabled = true;
+        bool MemRecallEnabled = true;
+        bool MemPlusEnabled = true;
+        bool MemMinusEnabled = true;
+        bool MemButtonEnabled = true;
```

Comment:
A couple comments here:
For consistency, private member variables should be prefixed with `m_`, and preferably begin with lowercase letters, similar to the flyout automation name variables below. I realize we haven't been perfectly consistent with this (such as the m_IsLastFlyout* variables above), but we should try to move towards consistency when possible.

Second, this change is adding 10 member variables just to track the enabled/disabled state of the various memory buttons for always-on-top mode. Can we find a way to reduce this or remove them altogether?